lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2382,6 +2388,18 @@ definitions:
       ues_connected:
         type: integer
         format: int32
+      gps_aititude:
+        type: string
+      vendor:
+        type: string
+      model_name:
+        type: string
+      rf_state:
+        type: boolean
+      sw_version:
+        type: string
+      uptime:
+        type: string
```

Comment:
Add some more details to the definitions here.
- Is there a description?
- What format would the `sw_version` come in?
- What format is `uptime` - this is likely to be standardized
- What does `rf_state` being `true` mean?

Reply:
Ideally, somebody reading the swagger definitions should know how to use the API.